This patch by Ralf Wildenhues <Ralf.Wildenhues@gmx.de> fixes
the "Hard fail" test case.

See http://thread.gmane.org/gmane.comp.sysutils.autoconf.bugs/6814
for details.

--- a/tests/autotest.at
+++ b/tests/autotest.at
@@ -271,7 +271,7 @@ AT_CHECK_AT_TEST([Hard fail],
    AT_CHECK([exit 99])],
   [], [1], [], [ignore], [],
   [AT_CHECK([grep '2 failed unexpectedly' micro-suite.log], [], [ignore])
-   AT_CHECK([grep ok micro-suite.log], [1])])
+   AT_CHECK([grep '^[[12]].*ok' micro-suite.log], [1])])
 
 AT_CHECK_AT_TEST([AT@&t@_FAIL_IF],
   [AT_FAIL_IF([:])
